app/controllers/organizations/foster_application_reviews_controller.rb

```diff
@@ -0,0 +1,4 @@
+class Organizations::FosterApplicationReviewsController < Organizations::BaseController
+  def index
+  end
+end
```

Comment:
Could we add the `layout "dashboard"` in these controllers?

Reply:
certainly can